src/SchemaPrintController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace SilverStripe\GraphQLDevTools;
+
+
+use GraphQL\Utils\SchemaPrinter;
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\GraphQL\Manager;
+
+class SchemaPrintController extends Controller
+{
+
+    public function index()
+    {
+        $key = $this->getRequest()->getVar('schema') ?: 'default';
+        $manager = Injector::inst()->createWithArgs(Manager::class, [$key]);
+        $manager->configure();
+        $schema = $manager->schema();
+        $content = SchemaPrinter::doPrint($schema);
+
+        return new HTTPResponse("<pre>$content</pre>", 200);
+    }
+
+    public function Link($action = null)
+    {
+        return 'dev/schemaprint';
```

Comment:
I'm wondering whether this URL isn't GraphQL specific enough - what about something like `dev/graphql-schema` or something?

Reply:
Lol, I actually gave this a lot of thought. Agree it's not ideal. Since most devs will be accessing this feature through the browser, I think the autocomplete experience is the challenge. We already have `dev/graphiql`, which is almost indistinguishable from `graphql`, so when you type `dev/grap..` you're likely to choose the wrong suggestion.

And a future release of graphql will use `dev/schema` to rebuild the schema.

I wonder if we need a new `dev/graphql` namespace?

Reply:
I like that idea =)